core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -192,14 +194,34 @@ class RDDDataDistribution private[spark](
     @JsonDeserialize(contentAs = classOf[JLong])
     val onHeapMemoryRemaining: Option[Long],
     @JsonDeserialize(contentAs = classOf[JLong])
-    val offHeapMemoryRemaining: Option[Long])
+    val offHeapMemoryRemaining: Option[Long]) {
+
+  def toLiveRDDDistribution(executors: scala.collection.Map[String, LiveExecutor])
+  : LiveRDDDistribution = {
```

Comment:
these added methods should probably be `private[spark]`

Reply:
Is this necessary ? Seems we've already has `private[spark]` scope for `RDDDataDistribution` and other objects which has toLiveXXX methods in `storeTypes`. So, those objects can't be constructed out of spark and these toLiveXXX methods won't be accessible, either.

Reply:
The classes themselves are public.  The `private[spark]` is just applied to the constructor (that's why its in an unusual position, that's the scala way of scoping the constructor).